sdk/healthinsights/azure-health-insights-radiologyinsights/README.md

```diff
@@ -84,26 +90,28 @@ Infer radiology insights from a patient's radiology report using a **synchronous
 - [SampleCriticalResultInferenceSync.java][ri_sync_sample]
 
 ```java com.azure.health.insights.radiologyinsights.inferradiologyinsightssync
-RadiologyInsightsInferenceResult riResults = radiologyInsightsClient.beginInferRadiologyInsights(createRadiologyInsightsRequest()).getFinalResult();
+RadiologyInsightsJob riJobResponse = radiologyInsightsClient.beginInferRadiologyInsights("job" + new Date().getTime(), createRadiologyInsightsJob()).getFinalResult();
```

Comment:
Nit: the previous variable name seems better. 

Also what is the new parameter for in this method?

Reply:
There was a POST to PUT change which requires now your own unique jobID to be set.
This also impacts the result we get back which is wrapped into a RadiologyInsightsJob object. The InferenceResult is nested in that one

Reply:
Usually GUID is better than the "current date".

Reply:
I replaced the current date with a UUID